Library/Statements/LetStatement.php

```diff
@@ -162,7 +164,17 @@ public function compile(CompilationContext $compilationContext)
                     break;
 
                 case 'static-property':
-                    $let = new LetStaticProperty();
+                    if (isset($assignment['operator'])) {
```

Comment:
@sergeyklay It could have been judged in `Zephir\statements\let\staticproperty`, but in order to be maintainable, it was put out separately, and add classes `Zephir\statements\let\staticpropertyadd`,`Zephir\statements\let\staticpropertySub`.